.travis/travis-deploy.sh

```diff
@@ -14,6 +14,10 @@ if [ "$TRAVIS_PULL_REQUEST" = "false" ] && [ "$TRAVIS_BRANCH" = "master" ]; then
     sed -i -e 's/\${revision}/'$VERSION'/' pom.xml */pom.xml
     git diff
     mvn -q deploy -DskipTests -Dcheckstyle.skip
+
+    # push jenkins plugin snapshot to https://repo.jenkins-ci.org/snapshots/fr/inria/repairnator
+    cd repairnator-jenkins-plugin
```

Comment:
we need to also build the plugin in `.travis/travis-run.sh`

Reply:
done :)